sim/shaman/shaman.go

```diff
@@ -63,7 +62,7 @@ const (
 	ClassSpellMask_ShamanHealingStreamTotem
 	ClassSpellMask_ShamanManaSpringTotem
 
-	ClassSpellMask_ShamanAll = ClassSpellMask_ShamanLast<<1 - 1
```

Comment:
Hmm I hadn't actually checked how `Last` was used here, just assumed it was right before `All`. Doesn't change existing tests but does change the meaning of the mask